tools/install-kustomize.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
+
+## https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh
```

Comment:
If this is a 1:1 copy of the upstream script, I'd suggest using `curl` to fetch the script in the `Dockerfile` instead of keeping a copy in `tools/`.